tasks/AccelerationController.cpp

```diff
@@ -26,6 +26,7 @@ bool AccelerationController::configureHook()
     if (! AccelerationControllerBase::configureHook()){
         return false;
     }
+    // Warning: By default, the storage orders in Eigen is column-major. So the thrusterMatrix will be gotten transposed.
```

Comment:
This helps a bit, but is not explicit as to why we transpose at all.

~~~
// As Eigen is row-major, the order of the `data` array in YAML configuration files appears
// to be wrong when loaded (i.e. [1, 2, 3, 4, 5, 6] is actually [[1, 4], [2, 5], [3, 6]]). We transpose
// here to get the same shape than in the YAML config file.
//
// Note that it will be confusing if written programatically
~~~

Moreover, I think you should check whether the matrix is ever used non-transposed. If it is not the case, do the transposition here once and for all and remove them in the rest of the code.

Reply:
Did it!